jellyfin_kodi/helper/utils.py

```diff
@@ -487,6 +488,13 @@ def set_addon_mode():
     LOG.info("Add-on playback: %s", settings("useDirectPaths") == "0")
 
 
+def strip_credentials(url):
```

Comment:
I want this function in the test suite, so I'd minimum need a list of URLs to test, both with and without credentials in them.
http(s), cifs/smb, nfs and (s)ftp come to mind, there are probably others I'm forgetting.

I can write the tests myself, but I need help gathering a diverse set of example URLs (that represent your real world application) – take what you have, but replace with [example/documentation IP addresses and domain names](https://alexwlchan.net/2018/ips-for-documentation/), and anonymize paths if needed (but keep the overall structure).

The above is a note for *anyone* using native mode, *especially* those with more esoteric setups.